examples/tf_cloudbuild_builder_simple/main.tf

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "cloudbuilder" {
+  source = "../../modules/tf_cloudbuild_builder"
+
+  project_id          = module.enabled_google_apis.project_id
+  dockerfile_repo_uri = google_sourcerepo_repository.builder_dockerfile_repo.url
+  # allow logs bucket to be destroyed
+  cb_logs_bucket_force_destroy = true
+}
+
+# CSR for storing Dockerfile
+resource "google_sourcerepo_repository" "builder_dockerfile_repo" {
```

Comment:
Can this be created in the cloud_builder module?

Reply:
@g-awmalik For now I kept the source repo outside the module to allow 3rd party VCS